practical/ProcessMining/group2/alphaplusminer/alpha_plus_miner.py

```diff
@@ -0,0 +1,479 @@
+from sortedcontainers import SortedSet, SortedDict
+from itertools import chain
+import numpy as np
+from graphviz import Digraph
+
+
+# Alpha Miner plus class
+class AlphaMinerplus:
```

Comment:
```suggestion
class AlphaMinerPlus(AlphaMiner):
```
Maybe for experimenting and comparing different miners later on, it would have made sense to implement a basic alpha miner first and then inherit from it to extend it for alpha+